services/how-to/avoid-rate-limiting.md

```diff
@@ -25,7 +25,7 @@ Requests for archive data are more expensive and are therefore subject to differ
 
 :::
 
-### Notice rate limiting behavior?
+### Notice rate limiting behavior
```

Comment:
```suggestion
### Notice rate limiting behavior?
```

This heading is phrased as a question, so doesn't make much sense without the question mark. We can change the wording to avoid the question mark in the future.